modules/images-allow-pods-to-reference-images-from-secure-registries.adoc

```diff
@@ -6,11 +6,19 @@
 [id="images-allow-pods-to-reference-images-from-secure-registries_{context}"]
 = Allowing pods to reference images from other secured registries
 
-The `.dockercfg` `$HOME/.docker/config.json` file for Docker clients is a Docker credentials file that stores your authentication information if you have previously logged into a secured or insecure registry.
+To pull a secured container from other private or secured registries, you must create a pull secret from your container client credentials, such as Docker or Podman, and add it to your service account. 
 
-To pull a secured container image that is not from {product-registry}, you must create a pull secret from your Docker credentials and add it to your service account.
+Both Docker and Podman use a configuration file to store authentication details when logging into secured or insecure registry:
```

Comment:
They use the file to store details when logging in, or to store details to log in?